stock_inventory_justification/models/stock_inventory_justification.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockInventoryJustification(models.Model):
+    _name = "stock.inventory.justification"
+    _description = "Inventory justification"
+
+    _sql_constraints = [
+        (
+            "unique_name",
+            "EXCLUDE (name WITH =) WHERE (active = True)",
+            "This stock inventory justification already exsits.",
```

Comment:
Please, let's try to fix the typo in this error message before merging, Overall, the module looks great, thanks for the effort!

Reply:
Done